src/coreclr/jit/lsraxarch.cpp

```diff
@@ -436,6 +436,19 @@ int LinearScan::BuildNode(GenTree* tree)
 
         case GT_XORR:
         case GT_XAND:
+            if (!tree->IsUnusedValue())
```

Comment:
```suggestion
            if (tree->IsUnusedValue())
```

Reply:
ah, the comment is wrong

Reply:
yes, never mind, i should have pointed to the wrong comment. Thanks!